CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -103,11 +103,19 @@ class DemographicsViewController: UIViewController {
     }
 
     @objc private func nextButtonPressed() {
-        /// Save onboarding information to user defaults, for user creation later.
-        for (key, value) in fieldValues {
-            userDefaults.set(value, forKey: key)
+        if let graduationYear = fieldValues[fieldMap[0]],
+           let major = fieldValues[fieldMap[1]],
+           let hometown = fieldValues[fieldMap[2]],
+           let pronouns = fieldValues[fieldMap[3]] {
+            NetworkManager.shared.updateUserDemographics(graduationYear: graduationYear,
+                                                  major: major,
+                                                  hometown: hometown,
+                                                  pronouns: pronouns,
+                                                  profilePictureURL: "").observe { successResponse in
+                                                    print("Update demographics success response \(successResponse)")
```

Comment:
I'm not a fan of this indentation. I don't think we do it like this usually, right?